### PR TITLE
Rename course

### LINKED ISSUE #16: Investigate renaming course to 'teamwork'-course
Investigate whether the scope of the course should be a bit broader, i.e. include also other insights into teamwork and reflect the Teamwork SIG scope.

### PR DESCRIPTION
**Pull request by svenvanderburg**
Fixes #26 

Rename course to 'Teamwork for Research Software'

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -28,10 +28,10 @@ kind: "lesson"
 # Magic to make URLs resolve both locally and on GitHub.
 # See https://help.github.com/articles/repository-metadata-on-github-pages/.
 # Please don't change it: <USERNAME>/<PROJECT> is correct.
-repository: escience-academy/lesson-agile-escience
+repository: escience-academy/teamwork-for-research-software-development
 
 # Email address, no mailto:
-email: "Agile Awareness Group - Agile-SIG <8afee2ac.esciencecenter.nl@emea.teams.ms>"
+email: "Teamwork course - Teamwork SIG <85f7ff15.esciencecenter.nl@emea.teams.ms>"
```
Comment (raar1) [inline _config.yml:34]: Is the Agile-SIG now called Teamwork-SIG? My MS Teams has not updated the group name yet, if so.

File: index.md [hunk 2]
```diff
@@ -4,14 +4,10 @@ root: .  # Is the only page that doesn't follow the pattern /:path/index.html
...
+This lesson is intended for people that create research software
+who would like to learn more about working in a team.
 
-This lesson has the following episodes:
-
-- The advantages and disadvantages of working in a team
-- What working in an agile manner is and why it is a good way of organizing a team.
-- The two major agile methodologies: Scrum and Kanban
-- How agile can be applied in a (e-)scientific setting
+FIXME: Expand description
```
Comment (raar1) [inline index.md:10]: Is there an issue for addressing this `FIXME`?

**Review by raar1 — APPROVED** [review]
I've left two questions, but otherwise looks good to me!

File: _config.yml (hunk 1 above)
Comment (svenvanderburg) [reply _config.yml:34]: That's the old Team, here is a [link](https://teams.microsoft.com/l/team/19%3ab0vlALQ1cKgmxye1YBUnbeTFh25ylqrbGg-CEOuZgAE1%40thread.tacv2/conversations?groupId=b41db130-c4bc-4d65-84d9-6b8e57e13416&tenantId=aa3aeacc-6307-42b2-ac05-787dd5c32574) to the new one.

File: index.md (hunk 2 above)
Comment (svenvanderburg) [reply index.md:10]: good point, there is now: #31

**Pull request merged**